db-scheduler-ui-frontend/public/config.json

```diff
@@ -0,0 +1 @@
+{"showTaskData":true,"showHistory":true}
```

Comment:
Skal historikk være opt in eller opt ut?

Reply:
Den configen brukes bare når man kjører frontend lokalt. Når man kjører example-app genereres en config.json basert på application.properties